asab/library/providers/zookeeper.py

```diff
@@ -63,6 +68,7 @@ async def read(self, path):
 
 		return node_data.decode('utf-8')
```

Comment:
Is this decode is needed? 
Last time your said  "Don't decode, a library should work with generic bytes blobs" (c) Ales

Reply:
it surprised me here also...